client/src/pages/Review/Review.tsx

```diff
@@ -1,25 +1,45 @@
-import { useState, useEffect, useMemo, useRef, useLayoutEffect } from "react";
-import { useParams } from "react-router-dom";
+import { useState, useEffect, useRef, useLayoutEffect } from "react";
+
 import { classnames } from "../../Utilities";
 import { Assessment, Course, Courses } from "../../logic/icsGen";
+import testState from "./data";
 
 import { IconButton } from "../../components/Button";
-
 import AppTopBar, {
   LeadingNavigation,
   TrailingIcon,
   Title,
   Subtitle,
 } from "../../components/AppTopBar";
 import CoursePanel from "./CoursePanel";
-
-import testState from "./data";
 import NavigationPanel from "./NavigationPanel";
 
 const Review = () => {
-  const { courseKey: courseKeyUrlParam } = useParams();
```

Comment:
This and all code blocks below relates to _url params_ shouldn't be removed. 

This gives the ability to redirect user to the specific course page if `course_id` is specified in the url.

Reply:
Yes, but this never did anything (except for with the test course) because reloading / redirecting reset app state anyway

Reply:
But this will be useful after the data is persistent in frontend (and PWA)

Reply:
True. Are we thinking localStorage for that? I don't know what a good option is

Reply:
https://reactrouter.com/en/main/hooks/use-before-unload looks good